metabolomics.yaml.lock

```diff
@@ -56,6 +56,19 @@ tools:
   owner: eschen42
   revisions:
   - 191a720488ce
+  - 582a8a42a93b
```

Comment:
We do not need the new-old version, isn't it? Please add only the newest version.